From 25a83178035e34bafb0de93dc170dc2ff32a2612 Mon Sep 17 00:00:00 2001
From: Sergio Pena <sergio.pena@cloudera.com>
Date: Fri, 23 Oct 2015 14:24:23 -0500
Subject: [PATCH 0357/1164] CLOUDERA-BUILD: CDH-33475: Add Hive itests/
 directory maven build into Hive
 pre-commit/post-commit tests

Change-Id: I96e61f629d45923c5226a1058a80b0024f5665d0
---
 cloudera/post_commit_hook.sh |    2 ++
 cloudera/pre_commit_hook.sh  |    2 ++
 2 files changed, 4 insertions(+)

diff --git a/cloudera/post_commit_hook.sh b/cloudera/post_commit_hook.sh
index a69dd52..88da81f 100755
--- a/cloudera/post_commit_hook.sh
+++ b/cloudera/post_commit_hook.sh
@@ -50,3 +50,5 @@ function get_regex_excluded_tests() {
 
 regex_tests=`get_regex_excluded_tests`
 mvn clean install -Phadoop-2 -Dtest.excludes.additional="$regex_tests"
+cd itests/
+mvn clean install -Phadoop-2 -DskipTests
diff --git a/cloudera/pre_commit_hook.sh b/cloudera/pre_commit_hook.sh
index e945ed2..a4c15b7 100755
--- a/cloudera/pre_commit_hook.sh
+++ b/cloudera/pre_commit_hook.sh
@@ -47,3 +47,5 @@ function get_regex_excluded_tests() {
 
 regex_tests=`get_regex_excluded_tests`
 mvn clean install -Phadoop-2 -Dtest.excludes.additional="$regex_tests"
+cd itests/
+mvn clean install -Phadoop-2 -DskipTests
-- 
1.7.9.5

